app/uk/gov/hmrc/singlecustomeraccountwrapperdata/controllers/ScaWrapperController.scala

```diff
@@ -46,12 +47,19 @@ class ScaWrapperController @Inject() (
     val httpUserAgent = request.headers.get(HeaderNames.USER_AGENT)
     val urBanners: List[UrBanner] =
       httpUserAgent.flatMap(urBannersConfig.getUrBannersByService.get(_)).getOrElse(List.empty)
+    val smartAppBannerUrlConfigs: List[SmartAppBannerUrlConfigs] =
+      httpUserAgent.flatMap(smartAppBannersConfig.getSmartAppBannersByService.get(_)).getOrElse(List.empty)
 
     val response = if (wrapperDataVersion == libraryVersion) {
       logger.info(
         s"[ScaWrapperController][wrapperData] Wrapper data successful request- version:$wrapperDataVersion, lang: $playLang"
       )
-      WrapperDataResponse(wrapperConfig.menuConfig(), wrapperConfig.ptaMinMenuConfig, urBanners)
+      WrapperDataResponse(
+        wrapperConfig.menuConfig(),
+        wrapperConfig.ptaMinMenuConfig,
+        urBanners,
+        smartAppBannerUrlConfigs
```

Comment:
The backend needs to pass the entire banner as well as the uri where it appears. So that only the backend is modified and not the library for any changes to the banner.